client/CMakeLists.txt

```diff
@@ -199,11 +204,11 @@ if (AUDIO_SDL)
 endif()
 
 if (APPLE)
-  qt5_import_plugins(freeciv21-client INCLUDE Qt5::QCocoaIntegrationPlugin Qt5::QSvgPlugin)
+  qt6_import_plugins(freeciv21-client INCLUDE Qt6::QCocoaIntegrationPlugin Qt6::QSvgPlugin)
 endif()
 
 if(EMSCRIPTEN)
-  qt5_import_plugins(freeciv21-client INCLUDE Qt5::QWasmIntegrationPlugin)
```

Comment:
If we are no longer supporting wasm in the CI, do we need to check for EMSCRIPTEN either?

Reply:
I'm not sure honestly. Removing it from the CI might have been a bold move, I just didn't want to spend time on getting it to work there.

Reply:
Will keep the code for now